### PR TITLE
feature/564 update header design to reflect prototype

### PR DESCRIPTION
**Pull request by superafroman**

### CODE REVIEW

File: src/lib/ui/template.ts [hunk 1]
```diff
@@ -161,6 +163,40 @@ function isFile(path: string) {
...
+				output += `<p>${line}</p>`
+			}
+		}
+		return output
+
+		//
+		// .filter(part => !!part)
+		// .map(part => `<p>${part}</p>`)
+		// .join()
+	}
```
Comment (m24murray) [inline src/lib/ui/template.ts]: Do we need this comment?

File: src/ui/component/Header.html [hunk 2]
```diff
@@ -2,8 +2,9 @@
     <div class="header-wrapper container">
         <div class="header-global">
             <div class="header-logo">
-                <a href="/" title="Go to the Learning Platform homepage" id="logo">
-                    Civil Service Learning
+                <a href="https://www.gov.uk" title="Go to the GOV.UK homepage" id="logo" class="content">
+                    <img src="img/gov.uk_logotype_crown_invert_trans.png" width="36" height="32" alt="">
```
Comment (m24murray) [inline src/ui/component/Header.html]: Do we need an alt tag?

File: src/lib/ui/template.ts (hunk 1 above)
Comment (superafroman) [reply src/lib/ui/template.ts]: Updated

File: src/ui/component/Header.html (hunk 2 above)
Comment (superafroman) [reply src/ui/component/Header.html]: There isn't one in the prototype - I think it's OK here to omit cos it's just a logo.

**Pull request merged**